model/peer/registry.yaml

```diff
@@ -13,3 +13,18 @@ groups:
           of the remote service. SHOULD be equal to the actual `service.name`
           resource attribute of the remote service if any.
         examples: "AuthTokenCache"
+        note: >
+          It MUST be possible to configure the peer service name in declarative configuration as follows:
```

Comment:
do we know which component would be responsible to use this configuration? This requirement should be actionable for this component. E.g. specialized span or log processor would match `network.protocol.address` to `peer` and populate `peer.service` on all spans.

In the current form, this requirement is only actionable for declarative config schema.